api/app/src/routes/webhook/apple.ts

```diff
@@ -17,9 +18,16 @@ routes.post(
     const metriportUserId = req.body.metriportUserId;
     const cxId = getCxIdOrFail(req);
     const payload = JSON.parse(req.body.data);
-    const mappedData = mapData(appleSchema.parse(payload));
 
-    processAppleData(mappedData, metriportUserId, cxId);
+    if (payload.error) {
+      capture.error(new Error(payload.error), {
```

Comment:
Wouldn't this result in the title at Sentry being the string representation of the error?

Maybe you share an example of how it looks like